presto-hive/src/main/java/com/facebook/presto/hive/security/SqlStandardAccessControl.java

```diff
@@ -186,6 +188,24 @@ public void checkCanCreateTable(ConnectorTransactionHandle transaction, Connecto
         }
     }
 
+    @Override
+    public void checkCanSetTableProperties(ConnectorTransactionHandle transactionHandle, ConnectorIdentity identity, AccessControlContext context, SchemaTableName tableName, Map<String, Object> properties)
```

Comment:
Should we implement method `checkCanSetTableProperties(...)` for `SystemTableAwareAccessControl` in hive as well?

Reply:
sure.